server/services/v1/billing/metronome.go

```diff
@@ -50,18 +52,52 @@ func NewMetronomeProvider(config config.Metronome) (*Metronome, error) {
 	return &Metronome{Config: config, client: client}, nil
 }
 
+func (m *Metronome) measure(ctx context.Context, operation string, f func(ctx context.Context) (*http.Response, error)) {
+	me := metrics.NewMeasurement(
+		metrics.MetronomeServiceName,
+		operation,
+		metrics.MetronomeSpanType,
+		metrics.GetMetronomeTags(operation))
+
+	resp, err := f(ctx)
+	me.RecordDuration(metrics.MetronomeLatency, me.GetTags())
+	if resp != nil {
+		defer resp.Body.Close()
+		me.IncrementCount(metrics.MetronomeResponseCode, me.GetTags(), strconv.Itoa(resp.StatusCode), 1)
+	}
+
+	var errCount int64
+	if err != nil {
```

Comment:
Should we log it as well?

Reply:
So its getting returned up the stack. We should be able to see a error log from upstream method call.

Reply:
sg